Makefile

```diff
@@ -15,6 +17,16 @@ $(CAIRO0_PROGRAMS_DIR)/%.json: $(CAIRO0_PROGRAMS_DIR)/%.cairo
 	@cairo-compile --cairo_path="$(CAIRO0_PROGRAMS_DIR)" $< --output $@ 2> /dev/null || \
 	docker run --rm -v $(ROOT_DIR)/$(CAIRO0_PROGRAMS_DIR):/pwd/$(CAIRO0_PROGRAMS_DIR) cairo cairo-compile /pwd/$< > $@
 
+%.mem: %.json
```

Comment:
The Giza benchmark reads `.memory` files, not `.mem` files. Or we change the extension here or in the benchmarks.

Reply:
Yes, I changed it everywhere, but I didn't push. Thanks for the review